.nvmrc

```diff
@@ -0,0 +1 @@
+v20.18.0
```

Comment:
It also has this.

> node -v > .nvmrc
> It will generate something like this inside your .nvmrc file:
> 
> v10.16.2
> Also using 10.16.2 without the v letter will work just fine.

 - approving the PR, need to be adopted if required.